modules/services/sql/variables.tf

```diff
@@ -10,6 +10,10 @@ variable "max_allocated_storage" {
   default = 100
 }
 
+variable "extra_security_group_ids" {
```

Comment:
Please add these new input variables to [sql/README](https://github.com/open-craft/terraform-scripts/blob/635890d9fb48661c6f531df494b1042bd23e5f1d/modules/services/sql/README.md#input)

Reply:
done :+1: